.rubocop.yml

```diff
@@ -1,6 +1,10 @@
+require:
+  - rubocop-minitest
+  - rubocop-performance
+  - rubocop-rake
```

Comment:
uh nice, that's a new one :)

Reply:
Unfortunately, it’s incompatible with Ruby 2.3, which we (for some reason) still support…